### ISSUE TITLE
Fix Storybook accessibility violations

### ISSUE THREAD

**Issue by NiklasBuchfink** [stagehq/ui#5]
Accessibility improvements to pass all Storybook violations:
- [ ] Elements must have sufficient color contrast
- [ ] IDs of active elements must be unique
- [ ] Images must have alternate text
- [ ] Links must have discernible text
